packages/chat-client/test/client/client.spec.ts

```diff
@@ -104,14 +138,16 @@ describe("ChatClient", () => {
 
     const invite: ChatClientTypes.PartialInvite = {
       message: "hey let's chat",
-      account: TEST_CLIENT_ACCOUNT,
+      account: `eip155:1${walletSelf.address}`,
```

Comment:
```suggestion
      account: `eip155:1:${walletSelf.address}`,
```
missing `:` based on the usage below?